network/network.go

```diff
@@ -132,21 +132,29 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	hosts := []string{}
 
+	start := time.Now()
+	c.logger.Log("level", "info", "message", fmt.Sprintf("connecting to kubernetes api to get %#q service info.", c.service))
```

Comment:
Can we make those logs more in line with what we have here? https://github.com/giantswarm/release-operator/blob/c1ddf954f28c2e43dc93a6e4e50f5b5062defa37/service/controller/release/resource/app/current.go#L25-L33

Reply:
Do you mean, I fact gathering with if , else if ...etc I feel in this case we have a bunch of returns after if statements so elses ,from my point of view does not look more readable. WDYT?